scheduler/api/thrift/scoot.thrift

```diff
@@ -71,9 +71,9 @@ struct JobDefinition {
   1: required list<TaskDefinition> tasks,
   2: optional JobType DEPRECATED_jobType,
   3: optional i32 defaultTaskTimeoutMs,
-  # Priority levels are defined in docs.
+  # the use of priority is deprecated.  Left in for api backwards compatibility
```

Comment:
will we be removing this at any point? also, should we rename it simlar to DEPRECATED_jobType?